avm/src/daemon/daemon.rs

```diff
@@ -0,0 +1,28 @@
+use once_cell::sync::OnceCell;
+use tokio::task;
+use tokio::time::{Duration, sleep};
+
+use crate::vm::run::run;
+
+pub static APP_ID: OnceCell<String> = OnceCell::new();
+pub static DEPLOY_KEY: OnceCell<String> = OnceCell::new();
```

Comment:
This is fine for now, but presumably we'll be able to replace this in the future during upgrades.

Reply:
Right, once we have VM ssh access

Reply:
It is possible to replace the value <https://doc.rust-lang.org/std/lazy/struct.OnceCell.html#method.take> but we'd have to have a mutable reference.

Reply:
> It is possible to replace the value https://doc.rust-lang.org/std/lazy/struct.OnceCell.html#method.take but we'd have to have a mutable reference.

This made me realize that I can just pass an immutable reference :)